pings.yaml

```diff
@@ -9,6 +9,19 @@
 ---
 $schema: moz://mozilla.org/schemas/glean/pings/2-0-0
 
+study-enrollment:
+  description: |
+    This ping is sent when the user has consented to the study.
+  include_client_id: false
+  send_if_empty: true
+  bugs:
+    - https://github.com/mozilla-rally/rally-study-01/issues/114
+  data_reviews:
+    - forthcoming
```

Comment:
I think you can safely use the previous data-review for this: it was already covering the case of the study-template being sent "when the study was installed into the study secure environment"